packages/core/lib/debug/printer.js

```diff
@@ -70,8 +71,17 @@ class DebugPrinter {
     //   glo: Global constants
     //   con: Contract variables
     //   loc: Local variables
+    // Note that this is a public variable and can be modified from outside.
     this.sectionPrintouts = new Set(["bui", "glo", "con", "loc"]);
     this.sections = ["bui", "glo", "con", "loc"]; //should remain constant
+
+    // numbers of instructions before and after the current instruction to be printed
+    // Note that this is a public variable and can be modified from outside.
+    this.instructionLines = { beforeLines: 3, afterLines: 3 };
+
+    // numbers of lines before and after the current line to be printed
```

Comment:
Hm, you previously had a comment here clarifying that this was just for `l` and `s`... not sure why you deleted that, that seems helpful.